css/layout.css

```diff
@@ -14,6 +15,7 @@
     display: flex;
     align-items: center;
     gap: var(--size-2);
+    padding-right: 10px;
```

Comment:
This madding is a bit mysterious -- maybe it's not needed?

Reply:
Without this padding on mobile view nav element align with the side of the screen.